package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@prisma/client": "2.24.0-dev.38",
```

Comment:
These were removed intentionally in this branch @williamluke4 as it can influence the available dependencies in the projects we are testing which can be a problem when looking for binaries etc.